src/backend/core/entitlements/resolvers/messages_storage_entitlement_resolver.py

```
@@ -0,0 +1,35 @@
+class MessagesStorageEntitlementResolver(EntitlementResolver):
+    """
+    Messages storage entitlement resolver.
+    """
+
+    resolve_level_prefix = "can_store"
+
+    def _resolve_entitlement(self, context, entitlement, metric):
+        """
+        Resolve the messages storage entitlement.
+        """
+        print('_resolve_entitlement', metric, entitlement.account_type)
```

Ruff (0.14.8)

`print` found

Remove `print`

(T201)